20190411/AGENDA.md

```diff
@@ -0,0 +1,15 @@
+# Sardana Follow-up Meeting - 2019/04/11
+
+To be held on 2019/04/11 at 14:00
+
+## Agenda
+
+1. Presentation
+	* Actual state of DSConfig (?)
+2. Urgent user problems/issues - Round table
+3. Overview of current developments / PR distribution
```

Comment:
I would like to talk about:
* SEP2
* CI for Windows #1078

Reply:
Added these topics, and also I think we should start working on a roadmap for Python3 support.